explorer/src/commitUtils.js

```diff
@@ -0,0 +1,93 @@
+// @flow
+
+type CommitData = {
+  // TODO improve variable names
+  fileToCommits: {[filename: string]: string[]};
+  commits: {[commithash: string]: Commit};
+  authors: string[];
+}
+
+type Commit = {
+  author: string;
+  stats: {[filename: string]: FileStats};
+}
+
+type FileStats = {
+  lines: number;
+  added: number;
+  deleted: number;
+}
+
+export function commitWeight(commit: Commit, filepath: string): number {
+  // hack - GitPython encodes renames in the filepath. ignore for now.
+  if (filepath.indexOf('=>') !== -1) {
+    return 0;
+  }
+  return Math.sqrt(commit.stats[filepath].lines);
```

Comment:
Fine for now, but—per our discussions, I thought that *this* was going to be log, and it would be the voting-power-as-a-function-of-cred that was radical?

Reply:
Yeah, log is probably a better choice. We'll play around with a few different weightFns (note that it's an argument, easy to swap)